frontend/model/state.js

```diff
@@ -751,8 +751,13 @@ sbp('okTurtles.events/on', CONTRACT_REGISTERED, (contract) => {
           //       period, then the distributionDate will get updated to the previous distribution date
           //       (incorrectly). That in turn will cause the Payments page to update and display TODOs
           //       before it should.
+          //       `distributionStarted` is different from getters.groupDistributionStarted
+          //       and it allows distributionDate can be updated
+          //       only from the 2nd distribution date when new distribution is started.
+          //       so this `distributionStarted` fixes the issue mentioned above
           const distributionDateInSettings = store.getters.groupSettings.distributionDate
-          if (oldPeriod && newPeriod && (newPeriod !== distributionDateInSettings)) {
+          const distributionStarted = reactiveDate.date > dateFromPeriodStamp(distributionDateInSettings)
```

Comment:
I'm not convinced this is needed. What happens if this change is removed?

Reply:
Hmm, OK. One thing though, I think the comment is wrong. You could use `store.getters.groupDistributionStarted(reactiveDate.date)`, it would be the same result. Can you make that change to DRY things?